packages/keyring-controller/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Removed
+
+- **BREAKING:** Remove `updateIdentities`, `syncIdentities`, `setSelectedAddress`, `setAccountLabel` from constructor options of `KeyringController` class.
```

Comment:
Do we also want to mention the methods that were changed along with the fact that that they no longer call these functions?

Reply:
```suggestion
- **BREAKING:** Remove `updateIdentities`, `syncIdentities`, `setSelectedAddress`, `setAccountLabel` callbacks, which previously updated `PreferencesController` state, from constructor options of `KeyringController` class. ([#3853](https://github.com/MetaMask/core/pull/3853))
  - Class methods `addNewAccount`, `addNewAccountForKeyring`, `createNewVaultAndRestore`, `createNewVaultAndKeychain`, `importAccountWithStrategy`, `restoreQRKeyring`, `unlockQRHardwareWalletAccount`, `forgetQRDevice` no longer directly updates `PreferencesController` state by calling the `updateIdentities` callback.
  - Class method `submitPassword` no longer directly updates `PreferencesController` state by calling the `syncIdentities` callback.
  - Class method `unlockQRHardwareWalletAccount` no longer directly updates `PreferencesController` state by calling the `setAccountLabel`, `setSelectedAddress, callbacks.
```

Reply:
@mcmire How does this look?

Reply:
@MajorLift Yeah, that looks good!